tests/elemental/container_validation.pm

```diff
@@ -44,6 +44,9 @@ sub run {
     my $img_filename = "elemental-$build-$arch";
     my $shared = '/var/shared';
 
+    # Set SELinux in permissive mode, as there is an issue with Enforcing mode and Elemental doesn't support it
+    assert_script_run("setenforce Permissive");
```

Comment:
This is not going to be a permanent change, right? On SLES 16 testing we're trying to avoid disabling SELinux except as workarounds, such as the ones in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/sles4sap/hana_install.pm#L129-L133

Reply:
For current version of Elemental it's a permanent change, as this version is in maintenance state. The newer version should be based on Unified Core but not planned to be released until 2026 I think (nothing written in stone yet). But of course for the next version it will have to fully support SELinux and I think it will be mandatory, as based on SLE16.